src/Adapter/Image/Uploader/ManufacturerImageUploader.php

```diff
@@ -80,8 +80,7 @@ private function generateDifferentSizeImages($manufacturerId)
 
         try {
             /* Generate images with different size */
-            if (isset($_FILES) &&
-                count($_FILES) &&
+            if (count($_FILES) &&
```

Comment:
count does perform an isset too ?

Reply:
$_FILES is a predefined variable : https://www.php.net/manual/en/reserved.variables.files.php